### ISSUE TITLE
PP filters: Need to improve the error message when the push fail by dependencies

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#18792]
Related to #17806 

Now when you use the PP filters we have multiple cases when the Pushing process fail by dependencies, we need to improve the error message in those cases, right now we are printing an error in the logs but in the bundle details we have a non specific error. 

## To recreate 
- Login as admin
- Create a new content type 
- Create a new piece of content 
- Try to push this piece of content using the just selected item filters

##Testing info
Tested on release-5.3.3 // Postgres // FF

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/enterprise-2.x/pull/733

**Comment by erickgonzalez** [comment]
New Error log that is shown in the UI (Bundle Detail) :
```
Failed to publish because an error occurred: Error Processing Contentlet in 
'/Users/erickgonzalez/Documents/dotcms/tomcat8/webapps/ROOT/assets/bundles/01ECTE0BNZTH5SVRT4SSQHERPP/live/d
emo.dotcms.com/1/0-content.c796aeee-a34c-4d79-a039-dedd92722a47.content.xml' with ID: '205a1762-6ba9-490c-
a229-729bcac1341e' because the ContentType: '37680f1a-824d-4b7a-98c6-c88c002d4209' does not exist
```

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.5 // Postgres // FF
